src/SAWScript/JavaExpr.hs

```diff
@@ -164,7 +166,7 @@ jeVarName = map dotToUnderscore . ppJavaExpr
         dotToUnderscore c = c
 
 asJavaExpr :: Term -> Maybe String
-asJavaExpr (asExtCns -> Just ec) = Just (ecName ec)
+asJavaExpr (asExtCns -> Just ec) = Just (Text.unpack (toShortName (ecName ec))) -- TODO?
```

Comment:
what's the TODO?

Reply:
Same as above, `toShortName` can have name collisions.